build/isc-bind9/files/empty.db

```diff
@@ -0,0 +1,8 @@
+$TTL 3h
+@ SOA @ nobody.localhost. 42 1d 12h 1w 3h
+        ; Serial, Refresh, Retry, Expire, Neg. cache TTL
+
+@       NS      @
+
+; Silence a BIND warning
```

Comment:
It might be better to say why this is necessary.
`All zones require at least one entry` or whatever the reason is.

Reply:
this I need to investigate a little... I will read up on this tomorrow. 
If you want to merge into extra, I am happy with this as it is just this one outstanding comment as an issue.

Reply:
yes, this `@       A       127.0.0.1` record is required, otherwise `named` fails with `zone xxxxxxxxxx has no address records (A or AAAA)`. comment has been updated and is ready to go.